python/hsfs/feature_group.py

```diff
@@ -1078,6 +1094,281 @@ def validate(
             ge_type=ge_type,
         )
 
+    def _get_feature_monitoring_configs(
```

Comment:
Are the new methods intended to be accessible by users? If so, why are they prefixed by `_`?

Reply:
Because we want to be able to merge in master but not make the UX public while we are stabilising it